README.md

```diff
@@ -18,6 +18,18 @@ Create `.auth.json` in the directory you will run `bin/sync` from:
 }
 ```
 
+### GitHub Enterprise
+
+To connect to a GHE you will need to add the GHE host to the `.auth.json` file above:
+
+```
+{
+  "username": "github-username",
+  "password": "github-password",
+  "enterprise_host": "github.your-GHE-company.com"
```

Comment:
Since GHE exposes the same API and all of the clients treat it exactly the same, it would be better to just have a single GH URL option that simply _defaults_ to `https://api.github.com/`.